index.js

```diff
@@ -76,7 +92,22 @@ const dostroy = (config) => async (req, res, next) => {
   const all = !config || Object.keys(config).length === 0
   const r = config && config.rudy ? config.rudy : RUDY_DEFAULT
   const rl = config && config.rateLimiting ? config.rateLimiting : RATELIMITING_DEFAULT
+  const dynamic = config && config.rateLimiting ? config.rateLimiting.useDynamicRateLimiting : USE_DYNAMIC_RATE_LIMITING_DEFAULT
+  const userActiveTimeout = dynamic && config.rateLimiting.userActiveTimeout ? config.rateLimiting.userActiveTimeout : USER_ACTIVE_TIMEOUT_DEFAULT
```

Comment:
We should somehow check that the user input is valid, so that we don't get unexpected errors or comparisons

Reply:
Yes, we will change to a flat config structure and check that userActiveTimeout is a number